README.md

```diff
@@ -1,7 +1,8 @@
-skopeo [![Build Status](https://travis-ci.org/containers/skopeo.svg?branch=master)](https://travis-ci.org/containers/skopeo)
+<!--- skopeo [![Build Status](https://travis-ci.org/containers/skopeo.svg?branch=main)](https://travis-ci.org/containers/skopeo)
 =
+--->
```

Comment:
I’d prefer completely dropping the `travis-ci…/svg` badge.

Please leave the “skopeo” heading text, so that the page actually starts with the name of the project (the logo does include the name, but it’s not that accessible to screen readers and the like). Or maybe set an `alt=` attribute on the image.

Reply:
I thought it looked a bit odd when the picture was resolved, and "Skopeo" was also there.  I have, however, added it as an alt tag, I like that thought.